.github/workflows/ci.yaml

```diff
@@ -27,31 +27,19 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Install cargo-aduit
-        uses: actions-rs/cargo@v1
```

Comment:
This action is unmaintained. Use cargo and leverage rust-toolchain.toml

Reply:
thanks